playwright/pages/profile/delegationPage.ts

```diff
@@ -172,3 +221,41 @@ export class revokeRights {
     }
   }
 }
+
+export class instantiateResource {
+  constructor(public page: Page) {}
+
+  async instantiateApp(appReportee: string) {
+    // Go to the app URL
+    await this.page.goto(process.env.APP_URL as string);
+    try {
+      const headingVisible = await this.page
+        .waitForSelector('role=heading[name="Hvem vil du sende inn for?"]', { timeout: 5000 })
```

Comment:
Dette er pirk, men vurder å bruke samme type for selectorer som du bruker nedenfor?

await page.getByRole('heading', { name: 'Hvem vil du sende inn for?' }).waitFor({ timeout: 5000 });